lavamoat/browserify/policy-override.json

```diff
@@ -62,37 +40,11 @@
         "crypto": true
       }
     },
-    "ethereumjs-wallet>randombytes": {
-      "globals": {
-        "crypto.getRandomValues": true
-      }
-    },
-    "web3": {
-      "globals": {
-        "XMLHttpRequest": true
-      }
-    },
-    "@metamask/controllers>web3": {
-      "globals": {
-        "XMLHttpRequest": true
-      }
-    },
     "react-dom": {
       "globals": {
         "HTMLIFrameElement": true
       }
     },
-    "react-devtools": {
```

Comment:
The `react-devtools` entries are no longer part of the production bundle, so they're not part of the policy anymore.